user-research/contributor-research/Introduction.md

```diff
@@ -0,0 +1,31 @@
+## Abstract:
+Knative is a vital tool in the open-source and cloud-native ecosystem, enabling the development of enterprise-level serverless and event-driven applications. However, its complexity poses challenges for both new contributors and maintainers. Providing guidance consumes valuable maintainer time, resulting in challenges when contributors leave after limited engagement.
+This research aims to understand why some individuals sustain active involvement in Knative while others contribute sporadically or disengage entirely. The ultimate goal is to identify aspects of the contributor experience that could be improved to enhance contributor retention and propose actionable recommendations to tackle these issues.
+
+## Research Goals:
+- Understand the current contributor journey within Knative.
+- Identify factors that contribute to contributors staying or leaving the Knative community.
+- Highlight areas of the contributor experience that can be improved to increase contributor retention and propose ways to improve those areas.
+
+## Objectives:
```

Comment:
What is the difference between research goals and objectives?

Reply:
Goals are like the bigger picture, while objectives are the smaller tasks to make those goals happen... 
For eg. to achieve the goal of understanding the current contributor journey within Knative (1st goal), there are a few potential objectives (1, 2 and to some extent, the 3rd one too).